### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by alishaikh9890**

### CODE REVIEW

File: src/components/BankDetails/BankDetails.styles.ts [hunk 1]
```diff
@@ -5,39 +5,44 @@ const bankDetailsStyles = {
...
+        '@media (max-width: 480px)': {
+            flexDirection:"column",  
+      },
     },
     text:{
         width:"35vh",
         margin:"0px"
-       
+    },
+    typo:{
```
Comment (amirmursal) [inline src/components/BankDetails/BankDetails.styles.ts:36]: rename this class

File: src/components/BankDetails/BankDetails.styles.ts [hunk 2]
```diff
@@ -5,39 +5,44 @@ const bankDetailsStyles = {
...
+        '@media (max-width: 480px)': {
+            width:"35vh",
+      },
+    },
+    divider1:{
```
Comment (amirmursal) [inline src/components/BankDetails/BankDetails.styles.ts:41]: rename this class

File: src/components/BankDetails/BankDetails.styles.ts [hunk 3]
```diff
@@ -5,39 +5,44 @@ const bankDetailsStyles = {
...
+        border: "2px solid gainsboro" 
+    },
+    divider2:{
```
Comment (amirmursal) [inline src/components/BankDetails/BankDetails.styles.ts:44]: this one as well

File: src/components/BankDetails/BankDetails.tsx [hunk 4]
```diff
@@ -24,48 +25,49 @@ const BankDetails = () => {
...
         </Grid>
 
         <Grid marginTop="5%" marginBottom="1%" display='flex' gap="5px"  >
-          <Button variant="contained"  >Back to Personal Details</Button>
+          <Button href='/addAgentDetails' variant="contained"  >Back to Personal Details</Button>
           <Button variant="contained">Next Page</Button>
         </Grid>
-
-
       </Box>
```
Comment (amirmursal) [inline src/components/BankDetails/BankDetails.tsx:69]: remove this extra line

File: src/components/PersonalDetails/PersonalDetails.styles.ts [hunk 5]
```diff
@@ -29,16 +25,26 @@ const personalDetailsStyles = {
...
+        },
     },
     text:{
         width:"35vh",
-        margin:"0px",
-
-       
+        margin:"0px", 
+    },
+    typo:{
```
Comment (amirmursal) [inline src/components/PersonalDetails/PersonalDetails.styles.ts:37]: same rename css classes

File: src/components/BankDetails/BankDetails.styles.ts (hunk 1 above)
Comment (alishaikh9890) [reply src/components/BankDetails/BankDetails.styles.ts:36]: Dome

File: src/components/BankDetails/BankDetails.styles.ts (hunk 2 above)
Comment (alishaikh9890) [reply src/components/BankDetails/BankDetails.styles.ts:41]: done

**Review by amirmursal — APPROVED** [review]

**Pull request merged**